s3stream/src/main/java/com/automq/stream/s3/S3Storage.java

```diff
@@ -590,7 +602,7 @@ public CompletableFuture<Void> forceUpload(long streamId) {
         final long startTime = System.nanoTime();
         CompletableFuture<Void> cf = new CompletableFuture<>();
         // Wait for a while to group force upload tasks.
-        forceUploadTicker.tick().whenComplete((nil, ex) -> {
+        forceUploadTicker.tick(enableBurstForceUpload && streamId != LogCache.MATCH_ALL_STREAMS).whenComplete((nil, ex) -> {
```

Comment:
It bypasses the effect of batch uploading, which is equivalent to triggering an upload each time a partition closes. In a scenario with 5000 partitions on a single machine, the effect of batching may be better.

If you think 500ms is too long, you can change the default value to 100ms

Reply:
yeah. when force upload the batch is still applied. but much lower from 500ms to 100ms.
we found that the more batch wait the more data may need to be upload. which also incr the close time.

AUTOMQ_BURST_UPLOAD_DELAY_TICK_TIME config change default to 100ms